modules/tinymce/src/themes/silver/main/ts/ui/dialog/IFrame.ts

```diff
@@ -34,11 +34,13 @@ const getDynamicSource = (initialData: Optional<string>): IFrameSourcing => {
 
 const renderIFrame = (spec: IframeSpec, providersBackstage: UiFactoryBackstageProviders, initialData: Optional<string>) => {
   const isSandbox = spec.sandboxed;
+  const isOpaque = spec.opaque;
 
   const attributes = {
     ...spec.label.map<{ title?: string }>((title) => ({ title })).getOr({}),
     ...initialData.map((html) => ({ srcdoc: html })).getOr({}),
-    ...isSandbox ? { sandbox: 'allow-scripts allow-same-origin' } : { }
+    ...isSandbox ? { sandbox: 'allow-scripts allow-same-origin' } : { },
+    ...isOpaque ? { class: 'tox-dialog__iframe tox-dialog__iframe--opaque' } : { class: 'tox-dialog__iframe' }
```

Comment:
As Andy mentioned on slack this should be using the `classes` property in the `dom` part of the alloy spec instead of setting it as an attribute. e.g:
```
dom: {
  tag: 'iframe',
  attributes,
  classes: isOpaque ? [ 'tox-dialog__iframe', 'tox-dialog__iframe--opaque' ] : [ 'tox-dialog__iframe' ]
},
```

Note: The above has some duplication so if possible we should probably create a base array and then extend it in the above instead.

Reply:
Thanks for clarifying that, now it makes sense